test_recommender_api.py

```diff
@@ -0,0 +1,29 @@
+from recommender_api import create_app
+import json
+import unittest
+
+class TestApi(unittest.TestCase):
+  """Test case for the flask api"""
+
+  def setUp(self):
+    self.app = create_app()
+    self.client = self.app.test_client()
+    self.data = {  # Represents a user
```

Comment:
En general evitaría comentarios que "ahorran" ser más explícitos en el código. Creo que mientras más explicativo sea el código y menos recurramos a comentarios que "parchan" la falta de autoexplicabilidad, mejor.

En cuanto a la estructuración general del test. Trataría de echarle una mirada a cómo ser consistente con los tests en Platanus en general. Se que no has visto otros tests, pero puedes usar de guía varios de los proyectos del github. Me pasa que creo que es bueno seguir la estructura del `set up` y de el test mismo separado, pero trataría de desgranar un poco la lógica del test mismo, y ver bien qué cosas son realmente setup y cuales realmente test. No se si por ejemplo el armado del path es parte del test en sí. 

También, no se si se puede, pero queda bien cuando en el mismo test hay una descripción o nombre de lo que se está probando. En rspec se usan mucho los `context` y mensajes en los `it` para describir el `expect` o `assert` en este caso.

Puede ser harta pega tratar de idear de 0 un estándar pero hacer cambios chicos en ese camino creo que es lo mejor por ahora y aportaría mucho!